conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+
+            <p>How a conflict is handled will depend on a number of factors including the type of complaint, severity, frequency, and contributing factors (for example, has anyone been involved in incidents before, is there a pattern?).</p>
+
+            <p>The procedure for addressing a conflict will include some or all of the following steps:</p>
+
+            <ol>
+                <li>Incident is reported.
+                    <ul>
+                        <li>Reporting can take on many forms including reporting the issue to a chair, ombudsperson, through an anonymous report, or a member of the team.</li>
+                    </ul>
+                </li>
+                <li>Incident is discussed, impact is shared, and first resolution is proposed.
+                    <ul>
+                        <li>For conflicts that have a clear path to resolution and low severity (for example, a harsh criticism in a meeting or inappropriate language), the focus should be on coming to a shared understanding of what happened and agreeing to a path to resolution (for example, an apology or editing a comment or minutes).</li>
+                    </ul>  
+                </li>
+                <li>
+                    ...
+                </li>
+            </ol>
+        </section>
+
+        <section id="ombudspeople">
+            <h2>The Role of the Ombudsperson</h2>
+
+            <p>Ombudspeople are advisors to participants who can provide guidance, advice, and support during the conflict resolution process. Ombudspeople are meant to be neutral third parties to any incident, and as such are required not to take sides or provide opinions on the indicidents they are involved in.</p>
+
+            <p>An Ombudsperson can advise or assist you with:</p>
+
+            <ul>
+                <li>How the Code of Conduct may apply in your situation</li>
+                <li>How to start the mediation process</li>
+                <li>How to raise a formal complaint</li>
+            </ul>
+
+            <p>TODO: More about the role of the ombudsperson and what they can and cannot do.</p>
+        </section>
+
+        <section id="mediation">
+            <h2>Mediation</h2>
+
+            <p>Mediation is most effective when used earlier in a dispute, before people become fixed in their position, as it provides a positive opportunity to resolve practical problems.</p>
+
+            <p>It empowers people to work together to resolve issues, and can therefore be less stressful and have a less negative impact on individuals. Mediation can also be successful in later stages after an investigation has concluded to help people rebuild relationships.</p>
```

Comment:
Less stressful than what?   I suggest shortening that first sentence: 
s/, and can therefore be less stressful and have a less negative impact on individuals//

Also add commas: "Mediation can also be successful in later stages, after an investigation has concluded, to help people rebuild relationships."